apps/mobile/src/screens/home/types/visit.ts

```diff
@@ -0,0 +1,16 @@
+export interface VisitByUserIdSerialized {
```

Comment:
Why did you recreate a type here? Couldn't you use the types coming from `db` package?